build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -216,8 +230,8 @@ service ActionCache {
 // `instance_name` is the empty path and the leading slash is omitted, so that
 // the `resource_name` becomes `uploads/{uuid}/blobs/{hash}/{size}`.
 // To simplify parsing, a path segment cannot equal any of the following
-// keywords: `blobs`, `uploads`, `actions`, `actionResults`, `operations` and
-// `capabilities`.
+// keywords: `blobs`, `uploads`, `actions`, `actionResults`, `operations`,
+// `capabilities` and `compressed-blobs`.
 //
 // When attempting an upload, if another client has already completed the upload
```

Comment:
Uploads of the same data may occur on both 'blobs' and 'compressed-blobs' concurrently.

Reply:
Added such a statement in the restructuring.